src/lib/backend-client.ts

```diff
@@ -21,9 +21,15 @@ export async function saveEnvironments(environmentsData: EnvironmentsData): Prom
   return await invoke("save_environments", { environmentsData });
 }
 
+export interface ProjectConfig {
+  cloud_token: string;
```

Comment:
I think especially in scope of this project, we'll probably want to run tests on the same cloud stack regardless of which project those tests belong to. So ideally `cloud_token` should be a part of app config, rather than project config 🤔

Reply:
we enter a dangerous zone discussing this topic 😂 

keeping it simple, with project-level we can have two projects, one targeting `grafana.com` and the other targets the dev one.
In here thou, it's implemented at project level just for time reasons, would make sense to revisit